pkg/terraform/terraform.go

```diff
@@ -127,20 +128,20 @@ func (m *Mixin) handleOutputs(outputs []Output) error {
 }
 
 // commandPreRun runs setup tasks applicable for every action
-func (m *Mixin) commandPreRun(step *Step) error {
+func (m *Mixin) commandPreRun(ctx context.Context, step *Step) error {
 	if step.LogLevel != "" {
 		os.Setenv("TF_LOG", step.LogLevel)
 	}
 
 	// First, change to specified working dir
 	m.Chdir(m.config.WorkingDir)
-	if m.Debug {
-		fmt.Fprintln(m.Err, "Terraform working directory is", m.Getwd())
-	}
+	//if m.Debug {
```

Comment:
Why was this commented out? Seems like the line just needed to be updated for DebugMode.